example/lib/widgets/main_button.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+
+import '../utils/styles.dart';
+import '../utils/dimensions.dart';
+
+class MainButton extends StatelessWidget {
+  const MainButton({
+    Key? key,
+    required this.onPressedFunction,
+    required this.children,
+  }) : super(key: key);
+  final void Function()? onPressedFunction;
+  final List<Widget> children;
```

Comment:
With the new implementation, List<Widget> is not necessary, and only add the others parameters.

Reply:
Changed!